src/flb_lib.c

```diff
@@ -664,14 +664,6 @@ int flb_start(flb_ctx_t *ctx)
     return 0;
 }
 
-int flb_loop(flb_ctx_t *ctx)
```

Comment:
is it necessary to remove this function ?, note that the usage of this function can be outside of the scope of the caller  fluent-bit.c, e.g: a custom application using Fluent Bit in library mode where is not associated to signals

Reply:
Oh, is it exposed as a library API ?
I won't remove such APIs.

Reply:
yeah, that should not be removed

Reply:
Sure. I removed the commit not to delete `flb_loop`